### PR TITLE
Clean up types for src/Server

### PR DESCRIPTION
**Pull request by spawnia**

### CODE REVIEW

File: src/Server/Helper.php [hunk 1]
```diff
@@ -159,7 +158,7 @@ public function parseRequestParams($method, array $bodyParams, array $queryParam
      *
      * @api
```
Comment (simPod) [inline src/Server/Helper.php:159]: do we know the array k,v type?

File: src/Server/ServerConfig.php [hunk 2]
```diff
@@ -57,45 +57,39 @@ public static function create(array $config = [])
         return $instance;
     }
 
-    /** @var Schema|null */
-    private $schema;
+    private ?Schema $schema = null;
 
     /** @var mixed|callable */
-    private $context;
+    private $context = null;
```
Comment (simPod) [inline src/Server/ServerConfig.php:63]: null is missing in `@var`

**Review by simPod — APPROVED** [review]

File: src/Server/Helper.php (hunk 1 above)
Comment (spawnia) [reply src/Server/Helper.php:159]: It is specified.

Generally, I did specify the type in return values when known and left it open in params where it does not matter.

File: src/Server/ServerConfig.php (hunk 2 above)
Comment (spawnia) [reply src/Server/ServerConfig.php:63]: Thanks, added.

**Pull request merged**